ratpack-core/src/main/java/ratpack/http/client/internal/RequestSpecBacking.java

```diff
@@ -51,6 +52,7 @@ public RequestSpecBacking(MutableHeaders headers, URI uri, ByteBufAllocator byte
     this.byteBufAllocator = byteBufAllocator;
     this.requestParams = requestParams;
     this.bodyByteBuf = byteBufAllocator.buffer(0, 0);
+    this.decompressResponse = false;
```

Comment:
I would say now is the time, before 1.0, to get this how we want.  Let's take advantage of not being concerned about backwards compatibility while we still can :)